src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,62 @@
+package homework_2.random_chars_table;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class RandomCharsTable {
```

Comment:
![image](https://user-images.githubusercontent.com/58872509/126908616-f8e238cc-c203-437f-8b5e-a2299b0aed35.png)

Reply:
![image](https://user-images.githubusercontent.com/58872509/126908623-79e5bcde-f4c6-4f48-a647-573e76fa15eb.png)

Reply:
![image](https://user-images.githubusercontent.com/58872509/126908637-52d1994c-00ee-4e1e-8eee-068e2d77251b.png)

Reply:
![Uploading image.png…]()

Reply:
Edge cases not handled. Please fix